.github/workflows/build.yaml

```diff
@@ -22,29 +24,72 @@ jobs:
       matrix:
         include:
         - { arch: aarch64, runs-on: macos-14 }
-        - { arch: x86_64, runs-on: macos-13 }
+        - { arch: x86_64, runs-on: macos-12 }
     runs-on: ${{ matrix.runs-on }}
-    timeout-minutes: 40
+    timeout-minutes: 120
     steps:
-    - run: mkdir -p build
-    - run: brew install filemonitor ninja
-    - run: brew install --ignore-dependencies lima # Don't pull in qemu
     - uses: actions/checkout@v4
       with:
         persist-credentials: false
-    - run: brew install --build-from-source ./qemu.rb
+        fetch-depth: 0
+    - id: version
+      name: Calculate version
+      shell: bash
+      run: |
+        version=$(git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
+        echo "VERSION=${version#v}" >>"$GITHUB_OUTPUT"
+    - name: Uninstall all brew packages and clear cache
```

Comment:
Would it be easier to just provide a different homebrew prefix? I'm wary of randomly uninstalling things that the rest of the stuff might need.

Reply:
I don't know, what is "the rest of the stuff"? This job just builds QEMU from source and then runs `lima-and-qemu.pl`. There is nothing else. And we explicitly install every other tool needed to do these.

I also don't think something like having multiple homebrew installations are going to work. Especially since we are running on a macOS version that homebrew really doesn't want to run on anymore, so each experiment takes hours.

Given that I've spent several days battling with homebrew about this, I'm kind of unwilling to spend extra effort on this unless there is an actual problem.